source/core/source/forge.js

```diff
@@ -1559,35 +1563,65 @@ var gca_forge = {
 
 		create : function() {
 			this.wrapper = document.createElement('div');
-			let aside, h2, section, table, tbody, tr, th;
+			let aside, h2, section;
+
+			const level_label = document.getElementById('icon_level').dataset.tooltip.split(',')[0].replace(/^\[+"|"$/g, '');
 
-			// First Table with prefixes
 			aside = document.createElement('aside');
-			aside.className = 'left';
+			aside.classList.add('left');
 			aside.style.width = 'calc(50% - 6px)';
 			h2 = document.createElement('h2');
-			h2.className = 'section-header';
-			h2.textContent = 'Prefix';
-			h2.appendChild(document.createTextNode(' '));
-			this.prefixNote = document.createElement('small');
-			h2.appendChild(this.prefixNote);
-			aside.appendChild(h2);
+			h2.classList.add('section-header');
+			h2.textContent = gca_locale.get('global', 'prefix');
+			let prefixNote = document.createElement('small');
+			h2.appendChild(prefixNote);
 			section = document.createElement('section');
 			section.style.display = 'block';
-			aside.appendChild(section);
-			table = document.createElement('table');
-			table.className = 'scroll-books-table';
-			section.appendChild(table);
-			tbody = document.createElement('tbody');
+			let table = document.createElement('table');
+			table.classList.add('scroll-books-table');
+			let thead = document.createElement('thead');
+			let th = document.createElement('th');
+			th.textContent = '#';
+			thead.appendChild(th);
+			th = document.createElement('th');
+			th.textContent = gca_locale.get('global', 'name');
+			thead.appendChild(th);
+			th = document.createElement('th');
+			let img = document.createElement('img');
+			img.alt = level_label;
+			img.title = level_label;
+			img.style.verticalAlign = 'middle';
+			img.src = gca_tools.img.cdn("img/ui/icon_level.gif");
+			th.appendChild(img);
+			thead.appendChild(th);
+			th = document.createElement('th');
+			th.title = gca_locale.get('global', 'details');
+			th.textContent = '🔎';
+			thead.appendChild(th);
+			table.appendChild(thead);
+			let tbody = document.createElement('tbody');
 			table.appendChild(tbody);
-			tr = document.createElement('tr');
-			tbody.appendChild(tr);
-			th = document.createElement('tr');
-			th.textContent = '...';
-			tr.appendChild(th);
+			section.appendChild(table);
+			aside.append(h2, section);
+
+			// First Table with prefixes
+			this.prefixNote = prefixNote;
 			this.prefixWrapper = tbody;
 			this.wrapper.appendChild(aside);
 
+			// "Show only not learned" label+checkbox for prefix
+			const hide_learned_label = document.createElement('label');
+			hide_learned_label.classList.add('section-header__checkbox');
```

Comment:
Minor comment: when custom class names are added, try to include the `gca` somewhere to avoid conflicts with the gladiatus codes.